packages/driver/src/cypress/command_queue.ts

```diff
@@ -238,9 +244,6 @@ export class CommandQueue extends Queue<$Command> {
       // reset the nestedIndex back to null
       this.state('nestedIndex', null)
 
-      // also reset recentlyReady back to null
-      this.state('recentlyReady', null)
-
```

Comment:
In Cypress 2017, `recentlyReady` was necessary to make `isReady` function (00920b1f9f93d0830003ac9198c3ad14e558489e). This function was removed when refactoring (d4e5b8ba5de1aa2c8f4474f2fc52c36bfe8c9730). 

But somehow this state survived. Removed it.